styles/nexteuropa_newsroom.css

```diff
@@ -544,11 +558,155 @@
   text-align: center;
 }
 
-.agendaPagination .date-form .container-inline-date .form-item input {
+.agendaPagination .container-inline-date .form-item input {
   text-align: center;
   color: #fff;
   background: #0065a2;
   cursor: pointer;
   border-radius: 10px;
 }
+
 /** Agenda presentation: end **/
+.newsroom-item--featured {
+  border-bottom: 2px solid #ececec;
+  margin-bottom: 2em;
+  padding-bottom: 1em;
+}
+
+.newsroom-item--featured .newsroom-item__title {
+  font-size: 150%;
+}
+
+.newsroom-item--featured .newsroom-item__featured {
+  display: inline-block;
+  background: #ffd617;
+  padding: 0.2em 1em;
+  color: #fff;
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 0.9em;
+}
+
+.newsroom-item--featured .newsroom-item__teaser {
+  font-size: 110%;
+}
+
+.newsroom-read-more,
+.newsroom_date {
+  font-style: italic;
+}
+
+.newsroom-read-more {
+  text-align: right;
+  display: block;
+  clear: both;
+}
+
+.newsroom-read-more::after,
+.newsroom_more::after,
+.read-full-text::after {
+  content: "";
+  width: 0;
+  height: 0;
+  border-top: 4px solid transparent;
+  border-bottom: 4px solid transparent;
+  border-left: 4px solid #0065a2;
+  display: inline-block;
+  margin-left: 0.5em;
+}
+
+.read-full-text {
+  font-weight: bold;
+  margin-top: 1em;
+}
+
+.newsroom_rss {
+  text-align: right;
+}
+
+.date-row {
```

Comment:
this selector seems too generic and can change other areas that you should not overwrite from nexteuropa_newsroom - could you take a look on why it was chosen instead of a more specific one?